src/store-modules/media-provider-store.js

```diff
@@ -0,0 +1,124 @@
+import {
+  FETCH_MEDIA_TYPE_PROVIDERS,
+  FETCH_MEDIA_PROVIDERS,
+} from './action-types'
+
+import {
+  SET_PROVIDER_FETCH_ERROR,
+  FETCH_MEDIA_PROVIDERS_END,
+  FETCH_MEDIA_PROVIDERS_START,
+  SET_MEDIA_PROVIDERS,
+  SET_PROVIDERS_FILTERS,
+} from './mutation-types'
+
+import previousImageProviders from '../data/existingImageProviders'
+import previousAudioProviders from '../data/existingAudioProviders'
+
+const AUDIO = 'audio'
+const IMAGE = 'image'
```

Comment:
Might be useful to make an enum of media that can be used globally as we'll see these constants being repeated frequently, moving forward.